[REF] test_server.py: Create server configuration file and copy attachment

moylop260 opened the pull request:
- Create server configuration file
   Usefully when you want start server manually without set --addons-path
 - Copy attachment from database template
 - Fix https://github.com/OCA/maintainer-quality-tools/issues/299

oscarolar commented:
:+1:

dreispt commented on travis/test_server.py:
```diff
@@ -276,7 +299,6 @@ def main(argv=None):
                      "-d", database,
                      "--stop-after-init",
                      "--log-level", test_loglevel,
-                     "--addons-path", addons_path,
```
Why not also pass the --data-dir option here, instead of using a conf file?

moylop260 replied:
To re-use when start server normally without a param extra

moylop260 commented:
ping

nhomar commented:
@moylop260 travis is red and coverage decrease, can you check that please?

moylop260 commented:
@nhomar 
Fixed https://github.com/vauxoo-dev/maintainer-quality-tools/commit/e59c9ea10680dd50be8f424d339360440b23f915
New version of flake8 inserted a new check, unrelated with this PR... but I fixed.

osvalr commented on travis/test_server.py:
```diff
@@ -225,6 +225,25 @@ def run_from_env_var(env_name_startswith, environ):
         subprocess.call(command, shell=True)
 
 
+def create_server_conf(data, version):
```
version parameter is not used, why leave it here?

osvalr commented:
:+1:

nhomar commented:
:+1:

moylop260 commented:
Can be merged?

moylop260 commented:
Thank you @max3903
